src/main/java/com/redhat/red/build/koji/model/json/BuildOutput.java

```diff
@@ -264,10 +265,10 @@ public Builder withMavenInfoAndType( ProjectVersionRef gav )
             return this;
         }
 
-        public Builder withNpmInfoAndType( String name, String version )
+        public Builder withNpmInfoAndType( NpmArtifactRef ref )
         {
             target.outputType = StandardOutputType.maven.name();
-            target.extraInfo = new FileExtraInfo( new NpmExtraInfo( name, version ) );
```

Comment:
What is the target.outputType assignment just above this line? Is that in the NPM method? Do we need a new output type in Koji to accommodate NPM artifacts?

Reply:
Good catch! This is a copy-paste err. Corrected.